eos-tech-support/eos-write-usb-internet

```diff
@@ -0,0 +1,243 @@
+ROAY70WqlJxDyEbD+ctTr0C20uoRnalc83CNzeJkKqtE4BjvRaqUnEPIRsP5y1OvQLbS6hGdqVzzcI3N
+4gAAAAA=
+BASE64
+)
+
+kiwix_library_xml=$(cat <<XML
+<library version="20110515">
+    <book
+        id="97e3f348-38f4-6787-b95d-1e4886bd803c"
+        path="../.kiwix/files/com.endlessm.encyclopedia.en-openzim-subscription/1.zim"
+        title="EndlessOS selection for English encyclopedia app"
+        description="EndlessOS selection for English encyclopedia app"
+        language="eng" creator="Wikipedia" publisher="Kiwix"
+        name="wikipedia_en_endless" flavour="_maxi"
+        tags="wikipedia;_category:wikipedia;_pictures:yes;_videos:no;_details:yes;_ftindex:yes"
+        date="2020-07-01" articleCount="50001" mediaCount="457288" size="6402513"
+        faviconMimeType="image/png"
+        favicon="iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAACXBIWXMAAAsSAAALEgHS3X78AAAFVklEQVRo3u1Ye6hlUxhfe+9zb3krEhOihm40MqOUyCMNkiJM5DZmTBh37fO65wwZUR7hPxnmDw1p8IeIlMdoGv7gDnFRZiQRNRFGHmXIPO7M3n7ft761z7f32Wcy5/4xV+1Vv9be31l7re/9fesYU41qVKMa1ahGNarx/xlp7OYkds/+3c9zl3GbYz7Aeyj0gGkr56gQ6YRjXlkgUs+jzPzEbA5o9Q5gk1rWTkQH0ZxAU0LrrbMHrnXeQ5jHfAje12LeDowLLRzKAolVB4hZ9+O/kQjnmJtwDJL5+xhXwsJlamqPy4HvoLg0vduk+O0tWRcN7UIkvTp8PnA1aDdhvg64ADhhgDChZlQznjHV+/1YYH3aAONNoA3E5jMIcJrsFyR2FsxjPgZ4GYfs5AMmgRYfQvgD9I+Bx4Er8H5YQeMRBWbSyKwa5nzdmhWgbWem3Z77gIcgTOCZ15nIZycf7Oxe4sbK1ZUb1c0oaSPtygGecS9IV9ABGvzbNmywDkxdkt7Wp/Gaej8PTGzm/UTr/B6bhSUKJIZDb9lSZTfE8sXUi4cWM0dMW/MucCVwLnARaMsxP4F5GtjJAnUEdV6/BfS7gHmK8VOB5/j3lriLNb8DE2pNLZc0CpkIwpBST4KSFmJehDUnqrgN8xaIzUZmyJrvKUPsJ+DHsKaDNR8Ae/kbb5nY/AWsxe+PsBt6F3QWW+/jSJiOJOtEBabHQGsAr+D5K6z7E9iD9z30jPOnQb9G1RLejKT5PF3FArwhh4wIPRSTRemyvkyzCPQH2ALWzLC221lwemtO4dAzFYMjEi+hcqHjQLOYp4BdLPgqQUdc2Fu9Lc/wmMz94P8jeNgqArwmB4Xik/naYHmtLkRHgcF1wvDuLMPErDXab5ozWYlVQT87dd/+xowpt8S534L+EnAv9l+OeRnWPol5h1g0AcZ0JppiV4AllLvUfCErHBzigEvxzYscEz5AnQtuBd7L/L6VaYwYmiSBgXOw5lUWsKOynTU/iAtejPnQAYXxev7O7fmg9r3HhEi4veTDI4DzgYeBL9hdvGmdxonxFT4D4X0xBH+bteXh4uSXLHachXax8qy5BTiyqCixOrlwTao6ecY37C2x8xZvgdOBGWFmHxZTqnsGeBbYhPdtzHRHac2a3WByQ+KKnj44UIqZzwHf4PX/8B4kgGN8L9AtMD1KSFRbkfbc2BU72s8J8E6uucKCGzKzdlUg3SmzY3qG64U19wNnFIuZBLzP64eDthr4iX1W1QJVzKhAbgSWUiEt7OfjLhJ39vtuIX4gyJtl1XgB8BTwqfjt18CHYgmLTc4qpD1feIJCAbuRi91kLjO9n1LsWLOSXa6pUrAT6Gfgefy+BJg3oGu4kD2lwwKsGdgP+VYXJo8GBFMW3IWsRHViQ1bNXQGjfL6khJnFoL/Ql4Va0rqQq7gstRpoAmt4bV0siSrff9HwgRPnb1C+HiQuAw3q7e/hPN7KNP43vruPaorvd3w3WxDkeOx5B+ibOE22lAt3C2jLHJtHc0rH4uKdwGR1QAtY77XgSa8BozL/CW/ezLISpclTCp1rXillwlhzMjDObmzNR8CPwA5Rxq/Sztys2/Zhr4befW7Nmr62NHmxubbY7+QU0+jrOgMJ/qDkrKMTy5lsTLf0cjma1d02kOcv6VIi6fFpyjxZDvdC0qWn3q8tBHjxxub7m0jajfLraVk3ekA3uFhdxGNzGZ5fB65S7lEr5vD/pJhmvxsnLrsF2X2A6ONu7VCXn6Il+ipnzzJz92+SJM7dd0Mp7+EwWj/of07py/3QWaEa1ahGNapRjWpU4+CNfwFKm+jnNVvQhQAAAABJRU5ErkJggg=="
+    />
+</library>
+XML
+)
+
+args=$(getopt -o t:h \
+    -l tmp-dir: \
+    -l debug \
+    -l help \
+    -n "$0" -- "$@")
+eval set -- "$args"
+
+tmp_dir=/tmp/eos-usb-internet-writer-tmp
```

Comment:
mktemp -d

Reply:
Is `/tmp` going to be big enough? On Endless OS it is a tmpfs, limited in size to the system's physical RAM.

Reply:
(If not, `/var/tmp` is disk-backed on Endless OS)

Reply:
Take into account this is the default and it's configurable, but I can change the default, sure.